source/security/index.html.md.erb

```diff
@@ -74,6 +76,16 @@ take care not to use any real card numbers. Read more about [testing](/testing_g
 
 GOV.UK Pay does not store full card numbers or CVV data for security reasons. This means you will not be able to search for transactions using card numbers. You can, however, search for the first six digits and the last four digits of a card.
 
+## Preventing fraudulent payments
+
+If you think you’re receiving fraudulent payments, you can:
+
+- [block users from paying with prepaid cards](https://docs.payments.service.gov.uk/optional_features/block_prepaid_cards/#block-prepaid-cards)
+- [delay capture of payments](https://docs.payments.service.gov.uk/optional_features/delayed_capture/#delayed-capture-of-payments) if you need time to do your own anti-fraud checks
+- check your payment service provider's (PSP's) security settings, or [contact us](https://docs.payments.service.gov.uk/support_contact_and_more_information/#contact-us) if your PSP is Stripe
+
+If your PSP is Worldpay, you can also block your users by their IP address. [Set up your WorldPay account to check IP addresses](http://support.worldpay.com/support/kb/gg/merchantadmininterface/Merchant%20Interface%20Guide.htm#5risk/risk_management.htm%3FTocPath%3DRisk%7C_____2), then [contact us](https://docs.payments.service.gov.uk/support_contact_and_more_information/#contact-us) and ask us to send your user's IP address to Worldpay during each payment.
```

Comment:
Can we also add a - if you have questions about fraud and how Pay may be able to help, get in touch...?

Reply:
We have Contact us in the nav, so I feel users should be able to make their way easily to contact info if they need it. There's a bit of a slippery slope where if we start adding prompts to contact us for the odd thing, there's no reason not to start adding it everywhere...